static/js/actions/ui.js

```diff
@@ -65,3 +65,18 @@ export const SET_USER_PAGE_DIALOG_VISIBILITY = 'SET_USER_PAGE_DIALOG_VISIBILITY'
 export const setUserPageDialogVisibility = bool => (
   { type: SET_USER_PAGE_DIALOG_VISIBILITY, payload: bool }
 );
+
+export const SET_SHOW_EDUCATION_DELETE_DIALOG = 'SET_SHOW_EDUCATION_DELETE_DIALOG';
```

Comment:
Yeah, and because they occupy the same screen space one will end up on top. Without the separate actions it was possible to have a situation where clicking the delete button on a work history entry and then confirming when the dialog appeared would cause one of the education entries to be deleted (on `/users`).